File: src/SocketBeControlled/MainRun.java
package SocketBeControlled;


import Utils.LogUtils;

public class MainRun {
    public static void main(String[] args) {
        try {
            LogUtils.initLog();
            LogUtils.logInfo("Main","LogInit!");
            System.out.println("123456");
            Client client = new Client("192.168.1.121",10086,"lzl471954654","Test");
            client.runClient();
        }catch (Exception e)
        {
            e.printStackTrace();
            LogUtils.logException("MainRun:Main",""+e.getMessage());
        }
    }
}

File: src/SocketBeControlled/MainRun.kt
package SocketBeControlled

import Utils.LogUtils

fun main(args: Array<String>) {
    val client = ControlledClient("127.0.0.1",10086,"Test")
    LogUtils.initLog()
    try{
        client.clientRun()
    }catch (e:Exception){
        LogUtils.logException("Main",""+e.message)
    }
}